src/api/app/components/diff_list_component.html.haml

```diff
@@ -0,0 +1,16 @@
+- diff_list.each_with_index do |(name, file_info), file_index|
+  - state = file_info['state']
+  .accordion.mb-2{ id: "diff-list-#{name.parameterize}" }
+    .accordion-item
+      %h2.accordion-header
+        %button.accordion-button.text-bg-light{ type: 'button', data: { 'bs-toggle': 'collapse',
+                                                                        'bs-target': "#diff-item-#{name.parameterize}" },
+                                                aria: { expanded: 'true', controls: "diff-item-#{name.parameterize}" } }
+          - if (old_file = file_info['old'])
+            = changed_filename(old_file['name'], name, state)
+          - else
+            = name
+          %span.badge.ms-1{ class: badge_for_state(state) }= state.capitalize
+      .accordion-collapse.collapse{ class: expand?(name, state) ? 'show' : '', id: "diff-item-#{name.parameterize}" }
+        - if (diff = file_info['diff'])
```

Comment:
So if there's a diff, we should render DiffComponent. It sounds like this belongs in [DiffComponent#render?](https://github.com/openSUSE/open-build-service/wiki/View-Components#conditional-rendering) since I don't see a case where that component would render when there is no actual diff.